provider/docker/docker.go

```diff
@@ -594,10 +594,25 @@ func (p *Provider) getIPAddress(container dockerData) string {
 
 	// If net==host, quick n' dirty, we return 127.0.0.1
 	// This will work locally, but will fail with swarm.
-	if "host" == container.NetworkSettings.NetworkMode {
+	if container.NetworkSettings.NetworkMode.IsHost() {
 		return "127.0.0.1"
 	}
 
+	if container.NetworkSettings.NetworkMode.IsContainer() {
+		dockerClient, err := p.createClient()
+		if err != nil {
+			log.Warnf("Impossible to get IP address for container %s, error: %s", container.Name, err)
```

Comment:
Could you replace `Impossible to get IP address for container...` by `Unable to get IP address for container...` :fr: